keps/sig-network/20190125-ingress-api-group.md

```diff
@@ -336,22 +345,23 @@ type IngressBackendHTTPHealthcheck struct {
 
```

Comment:
Can HTTPHealthcheck also support some additional queries? As in Liveness HTTP Check (https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request) you can specify Headers and you might also have different Return Codes being considered as healthy or not.

As an example, some Java applications take too long to deploy, and return 404 code when not ready. This might be used before putting the endpoint into the balancer.

Going forward, we can also think about different methods (GET, POST) but in my opinion, right now this would bring much bigger complexity that we need (as discussed we don't even know if this healthcheck will make into v1).

Reply:
We have remove healthchecking from the proposal for now.